test/tools/yulInterpreter/Interpreter.h

```diff
@@ -93,7 +93,7 @@ struct InterpreterState
 	util::h160 coinbase = util::h160("0x0000000000000000000000000000000077777777");
 	u256 timestamp = 0x88888888;
 	u256 blockNumber = 1024;
-	u256 difficulty = 0x9999999;
+	u256 prevrandao = 0x9999999;
```

Comment:
Ideally should be changed to a >2^64 number.

Reply:
Probably even ``< 2^64`` for ``<paris`` and ``> 2^64`` otherwise, but a bit annoying that this will then become evmversion dependent as well.

Reply:
As `prevrandao` is not supported by Hera (https://github.com/ewasm/hera/blob/master/src/wabt.cpp#L371) and ewasm will be dropped soon. I kept the `prevrandao` value to was it was before (i.e. `<= 2^64`), to still compatible with the current ewasm implementation and added a logic in `EVMInstructionInterpreter` to switch to the new value (i.e. `> 2^64`) when `evm >= paris`.

Reply:
Based on this comment https://github.com/ethereum/solidity/pull/13759#discussion_r1064898872 I changed the `InterpreterState` to contain both `difficulty` and `prevrandao` with their respective default values.